extensions/resource-deployment/src/services/notebookService.ts

```diff
@@ -58,22 +57,18 @@ export class NotebookService implements INotebookService {
 		return notebookPath;
 	}
 
-	/**
-	 * Get a file name that is not already used in the target directory
-	 * @param notebook source notebook file name
-	 * @param targetDirectory target directory
-	 */
-	getTargetNotebookFileName(notebook: string, targetDirectory: string): string {
-		const notebookFileExtension = '.ipynb';
-		const baseName = path.basename(notebook, notebookFileExtension);
-		let targetFileName;
-		let idx = 0;
-		do {
-			const suffix = idx === 0 ? '' : `-${idx}`;
-			targetFileName = path.join(targetDirectory, `${baseName}${suffix}${notebookFileExtension}`);
-			idx++;
-		} while (this.platformService.fileExists(targetFileName));
-
-		return targetFileName;
+	showNotebookAsUntitled(notebookPath: string): void {
+		let targetFileName: string = path.basename(notebookPath);
+		targetFileName = path.basename(targetFileName, '.ipynb');
+		const untitledFileName: vscode.Uri = vscode.Uri.parse(`untitled:${targetFileName}`);
```

Comment:
I'm a bit confused. Won't this break the 'Notebook-0', 'Notebook-1', etc logic? If you do file -> new notebook a number of times does the untitled name still count up?

Reply:
Again this is not general to notebooks, this part is specific to resource deployment extension and how we show big data notebooks.
Since we aren't saving them, it's user's discretion to save or over-write when prompted.